internal/app/query/query_service.go

```diff
@@ -135,7 +135,48 @@ func (s *queryService) QueryTicketIds(req *pb.QueryTicketIdsRequest, responseSer
 }
 
 func (s *queryService) QueryBackfills(req *pb.QueryBackfillsRequest, responseServer pb.QueryService_QueryBackfillsServer) error {
```

Comment:
what about checking req for nil?

Reply:
Here my grpc knowledge escapes me, but can a nil request reach this handler??

Reply:
Is this method supposed to be directly called by a grpc-client?

Reply:
Yes! Now i see your point... 👍